examples/webapp-management/01-AppServicePlan.sh

```diff
@@ -49,21 +49,21 @@ whpInfo2=`az app service plan get --name $whpName2`
 [ $(echo $whpInfo | jq '.sku.capacity' --raw-output) -eq $capacity ]
 
 printf "\n6. Get All app service plans by resource group: %s" "$groupName"
-plansByGroup=`az app service plan get --group $groupName`
+plansByGroup=`az appservice plan ls --group $groupName`
 [  $plansByGroup == *"$whpName"* ]
 [  $plansByGroup == *"$whpName2"* ]
 
 printf "\n7. Get All app service plans by location: %s" "$location2"
-plansByLocation=`az app service plan get -l $location2`
+plansByLocation=`az appservice plan ls -l $location2`
 [  $plansByLocation == *"$whpName2"* ]
 
 printf "\n8. Get All app service plans in a subscription."
-plansInSubscription=`az app service plan get`
+plansInSubscription=`az appservice plan ls`
 [  $plansInSubscription == *"$whpName"* ]
 [  $plansInSubscription == *"$whpName2"* ]
 
 printf "\n9. Remove app service plan: %s." "$whpName"
-az app service plan remove -n $whpName -g $groupName
+az appservice plan rm -n $whpName -g $groupName
 
 printf "\n10. Remove app service plan: %s." "$whpName2"
-az app service plan remove -n $whpName2 -g $groupName
+az appservice plan rm -n $whpName2 -g $groupName
```

Comment:
What is the distinction between appservice and webapp?  We should decide on one noun for this and use it throughout:

az appservice create
az appservice plan create
az appservice slot create

etc.